### ISSUE TITLE
Formatting Error in makePalRecordsConsistent

### ISSUE THREAD

**Issue by nkentNTIA** [Wireless-Innovation-Forum/Spectrum-Access-System#834]
Hey all,

The following currently occupies lines 208 and 209 of util.py:

```python
    pal_rec['license']['licenseAreaExtent'] = \
      'zone/census_tract/census/%d/%d' % (pal_census_year, pal_fips_code)
```

This produces an error if the user provides pal records that have string fips codes, which are necessary, as leading zeroes aren't allowed in Python ints or json numbers, and there are many census tracts for which the fips codes contain leading zeroes.

This could be fixed with the following: 

```python
    pal_rec['license']['licenseAreaExtent'] = \
      'zone/census_tract/census/{}/{}'.format(pal_census_year, pal_fips_code)
```

There are other pitfalls to using the old `%` syntax here, so i think format is preferable, but obviously this is up to the discretion of the TH dev team.

Cheers!

**Comment by kate-harrison** [comment]
Thanks! I put your fix into #836 after double-checking that it worked.

**Comment by nkentNTIA** [comment]
Thanks a bunch, Kate!

**Comment by kate-harrison** [comment]
Fixed in #836.
